docs/rules/array-bracket-newline.md

```diff
@@ -0,0 +1,243 @@
+# enforce line breaks after open and before close array brackets (array-bracket-newline)
+
+A number of style guides require or disallow line breaks inside of array brackets.
+
+## Rule Details
+
+This rule enforces line breaks after open and before close array brackets
+
+## Options
+
+This rule has either a string option:
+
+* `"always"` requires line breaks inside braces
+* `"never"` disallows line breaks inside braces
+
+Or an object option:
+
+* `"multiline": true` (default) requires line breaks if there are line breaks inside properties or between properties
+* `"minItems"` requires line breaks if the number of properties is more than the given integer
```

Comment:
That would also work. I just don't think "properties" is a good word here because it could cause confusion with properties in an object.